### PR TITLE
Bump pyro

### PR DESCRIPTION
**Pull request by ivaquero**

### CODE REVIEW

**Comment by Kapeli** [comment]
Dash uses the folder name as the unique identifier for the docset, so if you change the folder name from `pyro` to `Pyro`, everyone that has `pyro` installed will not get the update.

Please keep the folder name as `pyro`, and change the docset name within `docset.json` and the actual docset.

**Comment by ivaquero** [comment]
Alright, let me recover it.

File: docsets/pyro/docset.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
 {
     "name": "pyro",
```
Comment (Kapeli) [inline docsets/pyro/docset.json:2]: You can change the name to `Pyro` here, if you want.

Comment (ivaquero) [reply docsets/pyro/docset.json:2]: I'm okay with both.

**Comment by Kapeli** [comment]
Thank you!

**Pull request merged**